lib/database_cleaner/active_record/base.rb

```diff
@@ -15,7 +15,11 @@ def self.config_file_location
 
     class Base < DatabaseCleaner::Strategy
       def self.migration_table_name
-        ::ActiveRecord::SchemaMigration.table_name
+        if Gem::Version.new("6.0.0") <= ::ActiveRecord.version
```

Comment:
`ActiveRecord::Base.connection.schema_migration.table_name` can be used from Rails 6.0

https://github.com/rails/rails/pull/36439/files#diff-460f4e7973c5dd945c51d24df5b0173961190d3645f4e2585fd3003fa1fc0ff7